src/plugins/discover/public/application/components/sidebar/discover_field_search.tsx

```diff
@@ -226,6 +261,51 @@ export function DiscoverFieldSearch({ onChange, value, types }: Props) {
     );
   };
 
+  const footer = () => {
+    if (!showUnmappedFields && useNewFieldsApi) {
+      return null;
+    }
+    return (
+      <EuiPopoverFooter>
+        {showUnmappedFields ? (
+          <EuiFlexGroup>
+            <EuiFlexItem component="span">
+              <EuiSwitch
+                label={i18n.translate('discover.fieldChooser.filter.showUnmappedFields', {
+                  defaultMessage: 'Show unmapped fields',
+                })}
+                checked={values.unmappedFields}
+                onChange={handleUnmappedFieldsChange}
+                data-test-subj="unmappedFieldsSwitch"
+              />
+            </EuiFlexItem>
+            <EuiFlexItem component="span" grow={false}>
+              <EuiToolTip
+                position="right"
+                content={i18n.translate('discover.fieldChooser.filter.unmappedFieldsWarning', {
+                  defaultMessage:
+                    'Unmapped fields will soon be deprecated. We suggest keeping this switch off.',
```

Comment:
dear @gchaps & @andreadelrio FYI just to have a look at layout and wording
<img width="1105" alt="Bildschirmfoto 2021-01-29 um 19 30 58" src="https://user-images.githubusercontent.com/463851/106314800-8e657000-626a-11eb-8ef9-07820c904532.png">

Reply:
Typically, we use wording like this.  (Also, I'm suggesting to remove the second line).

> Unmapped fields are deprecated and will be removed in a future release.

If unmapped fields aren't yet deprecated, you can use this instead:

> Unmapped fields will be deprecated and removed in a future release.